src/mappings/Controller/mapping.ts

```diff
@@ -179,17 +180,12 @@ export function handleRegisterScheme(event: RegisterScheme): void {
   domain.handleRegisterScheme(avatar, token, reputation, event.params._scheme, paramsHash);
 
   // Detect a new organization event by looking for the first register scheme event for that org.
-  let isFirstRegister = store.get(
-    'FirstRegisterScheme',
-    avatar.toHex(),
-  );
+  let isFirstRegister = FirstRegisterScheme.load(avatar.toHex());
   if (isFirstRegister == null) {
     insertOrganization(event.address, avatar);
-    store.set(
-      'FirstRegisterScheme',
-      avatar.toHex(),
-      new Entity(),
-    );
+    isFirstRegister = new FirstRegisterScheme(avatar.toHex());
+    isFirstRegister.id = avatar.toHex();
```

Comment:
this line is redundant as the id is set be default at line 186.

Reply:
removed